lib/src/gauge.rs

```diff
@@ -72,3 +76,66 @@ impl StatisticSetGauge {
         })
     }
 }
+
+/// A GaugeDimensions is a group of dimensions to be used with a gauge.
+/// It is consumed when the gauge is created.
+#[derive(Debug, Default)]
+pub struct GaugeDimensions {
+    pub dimension_position: DimensionPosition,
```

Comment:
Why is this public? It's not useful to library consumers. Maybe you want to `impl From<GaugeDimensions> for DimensionPosition` instead?